translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"\t\t# Termina todos os recursos no caminho do diretório path/to/resources\n"
+"\t\tkubectl stop -f path/to/resources"
+
+#: pkg/kubectl/cmd/run.go:57
+msgid ""
+"\n"
+"\t\t# Start a single instance of nginx.\n"
+"\t\tkubectl run nginx --image=nginx\n"
+"\n"
+"\t\t# Start a single instance of hazelcast and let the container expose port "
+"5701 .\n"
+"\t\tkubectl run hazelcast --image=hazelcast --port=5701\n"
+"\n"
+"\t\t# Start a single instance of hazelcast and set environment variables "
+"\"DNS_DOMAIN=cluster\" and \"POD_NAMESPACE=default\" in the container.\n"
+"\t\tkubectl run hazelcast --image=hazelcast --env=\"DNS_DOMAIN=cluster\" --"
+"env=\"POD_NAMESPACE=default\"\n"
+"\n"
+"\t\t# Start a replicated instance of nginx.\n"
+"\t\tkubectl run nginx --image=nginx --replicas=5\n"
+"\n"
+"\t\t# Dry run. Print the corresponding API objects without creating them.\n"
+"\t\tkubectl run nginx --image=nginx --dry-run\n"
+"\n"
+"\t\t# Start a single instance of nginx, but overload the spec of the "
+"deployment with a partial set of values parsed from JSON.\n"
+"\t\tkubectl run nginx --image=nginx --overrides='{ \"apiVersion\": \"v1\", "
+"\"spec\": { ... } }'\n"
+"\n"
+"\t\t# Start a pod of busybox and keep it in the foreground, don't restart it "
+"if it exits.\n"
+"\t\tkubectl run -i -t busybox --image=busybox --restart=Never\n"
+"\n"
+"\t\t# Start the nginx container using the default command, but use custom "
+"arguments (arg1 .. argN) for that command.\n"
+"\t\tkubectl run nginx --image=nginx -- <arg1> <arg2> ... <argN>\n"
+"\n"
+"\t\t# Start the nginx container using a different command and custom "
+"arguments.\n"
+"\t\tkubectl run nginx --image=nginx --command -- <cmd> <arg1> ... <argN>\n"
+"\n"
+"\t\t# Start the perl container to compute π to 2000 places and print it "
+"out.\n"
+"\t\tkubectl run pi --image=perl --restart=OnFailure -- perl -Mbignum=bpi -"
+"wle 'print bpi(2000)'\n"
+"\n"
+"\t\t# Start the cron job to compute π to 2000 places and print it out every "
+"5 minutes.\n"
+"\t\tkubectl run pi --schedule=\"0/5 * * * ?\" --image=perl --"
+"restart=OnFailure -- perl -Mbignum=bpi -wle 'print bpi(2000)'"
+msgstr ""
+"\n"
+"\t\t# Inicia uma única instância de nginx.\n"
+"\t\tkubectl run nginx —image=nginx\n"
+"\n"
+"\t\t# Inicia uma única instância do hazelcast e expõe a porta 5701 do "
+"container.\n"
+"\t\tkubectl run hazelcast —image=hazelcast —port=5701\n"
+"\n"
+"\t\t# Inicia uma única instância do hazelcast e seta as variáveis de "
+"ambientente “DNS_DOMAIN=cluster” e “POD_NAMESPACE=default” no container.\n"
+"\t\tkubectl run hazelcast —image=hazelcast —env=“DNS_DOMAIN=cluster” —"
+"env=“POD_NAMESPACE=default”\n"
+"\n"
+"\t\t# Inicia uma instância replicada de nginx.\n"
+"\t\tkubectl run nginx —image=nginx —replicas=5\n"
+"\n"
+"\t\t# Dry run. Mostra os objetos da API correspondente sem criar elas.\n"
+"\t\tkubectl run nginx —image=nginx —dry-run\n"
+"\n"
+"\t\t# Inicia uma única instância de nginx, mas sobreescreve a spec do "
```

Comment:
I think the word "sobreescreve" here should be "sobrescreve". Searched if the current spelling exists, but couldn't find it.